.github/actions/upgrade-instances/action.yaml

```diff
@@ -26,12 +26,9 @@ runs:
         version: latest
 
     - name: Upgrade instances
-      shell: bash
+      shell: bash # do we need this?
```

Comment:
Hmm I added this since it doesn't work without it (not remember what the error exactly is...)

Reply:
Removed this. Let's try